### ISSUE TITLE
Make carousels accessible

### ISSUE THREAD

**Issue by allyceh** [usds/website#706]
Source: USDS.gov redesign questions & wishlist items in Slack

"Double-check the carousels for a11y if you intend to keep them, I cannot tab to them so I’m not altogether sure they are compliant."

**Comment by kategreenUSDS** [comment]
closing in favor of removing carousels as it's more accessible overall
